components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityMgtConstants.java

```diff
@@ -26,6 +26,29 @@ private IdentityMgtConstants(){}
     public static final String USER_IDENTITY_CLAIMS = "UserIdentityClaims";
     public static final String ERROR_CODE_DELIMITER = "-";
 
+    /**.
+     * DataType Constants
+     */
+    public enum DataTypes {
+
+        STRING("string"),
+        INTEGER("integer"),
+        FLOATING("float"),
+        BOOLEAN("boolean");
+
+        private final String value;
+
+        DataTypes(String values) {
+            this.value = values;
+        }
+
+        public String getValue() {
+            return this.value;
+        }
+
+    }
+
+
```

Comment:
Shall we remove the unnecessary new lines?

Reply:
Fixed in commit 8032899333352cf8a310623b49856dfb99ec9767